SSA/Projects/InstCombine/LLVM/Semantics.lean

```diff
@@ -229,7 +251,7 @@ def sdiv {w : Nat} (x y : IntW w) (flag : ExactFlag := {exact := false}) : IntW
   let y' ← y
   let exact := flag.exact
   let Exact? : Prop := exact ∧
-    (x'.toInt % y'.toInt != 0)
+    (x'.smod y' ≠ 0)
```

Comment:
I guess we can clean this up as well by folding things together as suggested above.